tracing/tracing/src/main/java/io/helidon/tracing/ExtendedTracerConfigBlueprint.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.tracing;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+
+import io.helidon.builder.api.Option;
+import io.helidon.builder.api.Prototype;
+import io.helidon.common.configurable.Resource;
+
+/**
+ * Common settings for tracers including settings for span processors and secure client connections.
+ */
+@Prototype.Blueprint
+@Prototype.Configured
+interface ExtendedTracerConfigBlueprint {
+
+    /**
+     * Service name of the traced service.
+     *
+     * @return service name
+     */
+    @Option.Configured("service")
+    String serviceName();
+
+    /**
+     * URI for the collector to which to send tracing data.
+     *
+     * @return tracing collector URI
+     */
+    Optional<URI> collectorUri();
```

Comment:
This can be `@Configured` as well, URI is supported

Reply:
Changed.